### PR TITLE
CPS-???? - Coin Selection Including Native Tokens

### PR DESCRIPTION
**Pull request by SIDANWhatever**
This seems a long withstanding issue (#232) where most offline library we got now still using sub-optimal coin selection strategy which might lead to inefficiency transaction building process.

Raising this CPS in an attempt to attract community's attraction to solve this issue with combined intelligence.

### CODE REVIEW

File: CPS-0002/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+CPS: 2
```
Comment (rphair) [inline CPS-0002/README.md:2]:
```suggestion
CPS: ?
```

File: CPS-0002/README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+Title: Coin Selection Approach for Post Native Tokens Era in Cardano
```
Comment (rphair) [inline CPS-0002/README.md:3]:
```suggestion
Title: Coin Selection Including Native Tokens
```
If author & editors are in agreement that this is more specific and less redundant ("era", "Cardano") then let's please change the document title to the current PR title.

**Review by rphair — COMMENTED** [review]
I've edited the title down for specificity and to remove redundancy.

Note there was a proposed CPS-0002 which in my opinion still represents an unresolved issue which could be reactivated: https://github.com/cardano-foundation/CIPs/pull/374

In any case, when skipping over numbers due to deprecations or abandonments we don't go back and fill in the missing numbers, but would rather choose the next highest number in sequence after the last assignment.

In general I think it's good to have this as a CPS so thanks @SIDANWhatever for your submission & looking forward to discussing this in the community soon.

File: CPS-0002/README.md (hunk 2 above)
Comment (rphair) [reply CPS-0002/README.md:3]: p.s. also @SIDANWhatever for fixing little things like this we recommend that your PR branch be set to "Allow edits by maintainers" (the general default when submitting a PR, and helpful to keep track of who recommended the changes behind each commit).

Comment (SIDANWhatever) [reply CPS-0002/README.md:3]: > p.s. also @SIDANWhatever for fixing little things like this we recommend that your PR branch be set to "Allow edits by maintainers" (the general default when submitting a PR, and helpful to keep track of who recommended the changes behind each commit).

Seems its only work for PR from feature branch, should I close this PR and create a new one?

Comment (rphair) [reply CPS-0002/README.md:3]: no need to create a new PR... you should be able to extend the permissions by following these instructions (i.e. when you view this PR's branch in the repository which pushed the pull request) 🤓 https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests

**Pull request closed without merging**